docs/cephfs-snap-clone.md

```diff
@@ -19,6 +19,18 @@
   be a `volumesnapshotclass` object present in the cluster
   for snapshot request to be satisified.
 
+  - To install snapshot controller and Beta snapshot CRD
+
+    ```console
```

Comment:
please mention about setting the snapshotter version as it will be useful.

change Beta CRD to CRD as we don't need to change anything once we move to a stable version

Reply:
done now.